examples/03_datetime_encoder.py

```diff
@@ -1,89 +1,112 @@
+# on the location, date and time of measurement.
+
 import pandas as pd
 
 data = pd.read_csv(
-    "https://raw.githubusercontent.com/pandas-dev/pandas/main/doc/data/air_quality_no2_long.csv"  # noqa
+    "https://raw.githubusercontent.com/pandas-dev/pandas"
+    "/main/doc/data/air_quality_no2_long.csv"
 )
+# Extract our input data (X) and the target column (y)
 y = data["value"]
 X = data[["city", "date.utc"]]
+
 X
 
 ###############################################################################
-# Encoding the data to numerical representations
-# ----------------------------------------------
+# Encoding the features
+# .....................
+#
+# We will construct a |ColumnTransformer| with |make_column_transformer|.
+# We'll want to encode the city names with a |OneHotEncoder|, and the date
+# with a |DatetimeEncoder|.
 #
-# Encoders for categorical and datetime features
-# ..............................................
+# During the instantiation of the |DatetimeEncoder|, we specify that we want
+# to extract the day of the week, and that we don't want to extract anything
+# finer than minutes. This is because we don't want to extract seconds and
+# lower units, as they are probably unimportant.
+
 from sklearn.preprocessing import OneHotEncoder
 
 from skrub import DatetimeEncoder
 
-cat_encoder = OneHotEncoder(handle_unknown="ignore")
-# We encode dates using the day of the week as it is probably relevant,
-# but no longer than minutes: we are probably not interested in seconds
-# and below
-datetime_encoder = DatetimeEncoder(add_day_of_the_week=True, extract_until="minute")
-
 from sklearn.compose import make_column_transformer
 
-datetime_columns = ["date.utc"]
-categorical_columns = ["city"]
-
 encoder = make_column_transformer(
-    (cat_encoder, categorical_columns),
-    (datetime_encoder, datetime_columns),
+    (OneHotEncoder(handle_unknown="ignore"), ["city"]),
+    (DatetimeEncoder(add_day_of_the_week=True, extract_until="minute"), ["date.utc"]),
     remainder="drop",
 )
 
-###############################################################################
-# Transforming the input data
-# ...........................
-# We can see that the encoder is working as expected: the date feature has
-# been replaced by features for the month, day, hour, and day of the week.
-# Note that the year and minute features have been removed by the encoder
-# because they are constant.
-X_ = encoder.fit_transform(X)
+X_enc = encoder.fit_transform(X)
 encoder.get_feature_names_out()
+
 ###############################################################################
-X_
+# We can see that the encoder is working as expected: the "date.utc" column has
+# been replaced by features extracting the month, day, hour, and day of the
+# week information.
+#
+# Noting the year and minute features are not present, this is because they
+# have been removed by the encoder as they are the same for the whole feature.
```

Comment:
```suggestion
# Note the year and minute features are not present, this is because they
# have been removed by the encoder as they are constant the whole period.
```